docs/core/howto/defer-intro.rst

```diff
@@ -315,6 +315,10 @@ Otherwise, ``g`` will be passed the asynchronous equivalent of the return value
 Inline callbacks - using 'yield'
```

Comment:
If we want folks to use `async def` instead of `@inlineCallbacks` the sections should be swapped so that the suggested option is first.

Reply:
done